src/main/kotlin/no/nav/medlemskap/regler/v2/common/Resultat.kt

```diff
@@ -0,0 +1,13 @@
+package no.nav.medlemskap.regler.v2.common
+
+enum class Svar {
+    JA, NEI, UAVKLART
```

Comment:
Var vi ikke over på bare JA/NEI? Eller er UAVKLART her fordi vi nå bare har ett nivå med abstraksjon?

Reply:
Vi bruker fortsatt uavklart i svaret vårt, så den må være med et sted. Spørsmålene våre er JA/NEI, men konklusjonen er fortsatt UAVKLART. Så det er et forsøk på å ikke ha for mange abstraksjonslag

Reply:
Ja, riktig, jeg skjønner problemstillingen. Helt greit for meg det valget du har tatt der. :)